lib/ui/shared/preferences/preferenceswidgets/theme/theme_viewmodel.dart

```diff
@@ -36,8 +40,10 @@ enum cleanThemes { aubergine, versatile, aubergine1, aubergine2 }
 
 class ThemeViewModel extends BaseViewModel {
   final log = getLogger("ThemeViewModel");
-
   final _themeService = locator<ThemeService>();
+  final _localStorage = locator<LocalStorageService>();
+  final _themeStorageKey = 'themeSetting';
```

Comment:
This too, all keys should be setup in the localStorage file

Reply:
done